fedramp-integrated-inventory-workbook/.vscode/launch.json

```diff
@@ -0,0 +1,41 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Weird, not legal json but I assume vscode is reading this properly?

Reply:
To add on, is VSCode even needed? Maybe delete this directory?

Reply:
Removed lots of unused files, including some of the license ones and the .vscode directory.